packages/backend/src/ee/services/AiAgentService.ts

```diff
@@ -3537,4 +3531,18 @@ export class AiAgentService {
 
         return this.aiAgentModel.getArtifact(artifactUuid, versionUuid);
     }
+
+    async appendInstruction(
+        user: SessionUser,
+        agentUuid: string,
+        instruction: string,
+    ): Promise<string> {
+        // Check user has access to the agent
+        await this.getAgent(user, agentUuid);
```

Comment:
might as well pass the third arg `projectUuid`

Reply:
nice, done